homeassistant/components/motionmount/entity.py

```diff
@@ -42,12 +42,45 @@ def __init__(self, mm: motionmount.MotionMount, config_entry: ConfigEntry) -> No
                 (dr.CONNECTION_NETWORK_MAC, mac)
             }
 
+    @property
+    def available(self) -> bool:
+        """Return True if the MotionMount is available (we're connected)."""
+        return self.mm.is_connected
+
+    def update_name(self) -> None:
+        """Update the name of the associated device."""
+        # Check whether we need to update our name
+        if self.device_info and self.device_info[ATTR_NAME] != self.mm.name:
```

Comment:
Since we set `_attr_device_info` unconditionally in `MotionMountEntity.__init__`, there's no need to check if it's truthy here.
If mypy complains, instead do like this:
```py
    if TYPE_CHECKING:
        assert self.device_info
```